src/codecs/webp/vp8.rs

```diff
@@ -865,6 +871,50 @@ pub struct Frame {
     sharpness_level: u8,
 }
 
+impl Frame {
+    /// Chroma plane is half the size of the Luma plane
+    fn chroma_width(&self) -> u16 {
+        (self.width + 1) / 2
+    }
+
+    fn chroma_height(&self) -> u16 {
+        (self.height + 1) / 2
+    }
+
+    /// Converts the 4:2:0 YUV vectors to an rgb vector
+    pub fn to_rgb_vec(&self) -> Vec<u8> {
```

Comment:
You could probably just fold this into the `into_reader` function, since that's the only place this is called. Could even be a simple as:
```rust
fn into_reader(self) -> ImageResult<Self::Reader> {
    let mut data = vec![0; self.ybuf.len() * 3];
    self.fill_rgb(&data);
    Ok(WebpReader(Cursor::new(data), PhantomData))
}
```

Reply:
Seems like a good idea.